### PR TITLE
Viewport Meta Tag Refactored #570

### PR DESCRIPTION
**Pull request by dmbeer**
Refactored viewport meta tag to be customisable and to take the parameters required to create the correct viewport tags.

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/html/MobileViewportMetaTag.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import org.apache.wicket.IGenericComponent;
+import org.apache.wicket.markup.ComponentTag;
+import org.apache.wicket.markup.html.WebMarkupContainer;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+
+/**
+ * A special {@link MetaTag} that adds a TODO
+ *
+ * @author miha
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/html/MobileViewportMetaTag.java]: You keep pretending to be `miha` :-)

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/html/MobileViewportMetaTag.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        return name.getObject();
+    }
+
+    public String getWidth() {
+        return width;
+    }
+
+    public void setWidth(String width) {
+        this.width = "width=" + width;
+        this.setModelObject(this.width);
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/html/MobileViewportMetaTag.java]: Why it is kept both as member field and as model object?

**Comment by dmbeer** [comment]
@martin-g update to reflect suggestions. The widht is now consistent with the others. The reason for setting independately means that value can be retrived if they want them. I am happy to change this though to just pass the value. Not having a member as such just specific methods.

**Comment by martin-g** [comment]
@dmbeer I've made some extra changes. Please review and tell me if you don't like it.
Thanks for the contribution!

**Comment by dmbeer** [comment]
@martin-g I can see why it was changed to WebComponent, but changing userScale to boolean just adds more processing and conversion. Also if going to have a builder process I would move it a separate method.

**Comment by martin-g** [comment]
'user-scalable' could have just two values: `yes` and `no`. So a boolean looks much better to me. Otherwise the application developer could set some invalid value
The processing time should be neglectable.

**Pull request merged**